frame/support/src/lib.rs

```diff
@@ -127,7 +127,7 @@ pub const LOG_TARGET: &str = "runtime::frame-support";
 pub enum Never {}
 
 /// A pallet identifier. These are per pallet and should be stored in a registry somewhere.
-#[derive(Clone, Copy, Debug, Eq, PartialEq, Encode, Decode, TypeInfo, MaxEncodedLen)]
+#[derive(Clone, Copy, Eq, PartialEq, Encode, Decode, TypeInfo)]
 pub struct PalletId(pub [u8; 8]);
```

Comment:
this is a public api of frame.
why you removing MaxEncodedLen derive?
Your assumption that it is not used by anyone and anything expect your use case?

Reply:
Sorry, I needed to set the context. I've added this in the prev PR where I needed the PalletId, but now we decided to remove it from the namespace `Pallet(PalletId)` => `Pallet`. That's why I simply reverting the prev changes